scripts/google/covid_mobility/TestCovidMobility.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from covidmobility import covid_mobility, csv_row_to_obj
+
+
+# NOTE: More testing will be added soon.
+
+
+class TestStringMethods(unittest.TestCase):
+    def test1_output(self):
+        # Get the output file.
+        covid_mobility('./test1.csv')
+        # Get the content of the expected output.
```

Comment:
I think you flipped the two comments. The first one is actual/written output.

Reply:
fixed.